.appveyor.yml

```diff
@@ -19,7 +55,7 @@ install:
   - '%PYTHON_HOME%\Scripts\easy_install --version'
   - '%PYTHON_HOME%\Scripts\pip --version'
   - '%PYTHON_HOME%\Scripts\tox --version'
-  - ps: (new-object net.webclient).DownloadFile('https://osf.io/rmfb7/download', 'my_test_solver.exe')
+  - ps: (new-object net.webclient).DownloadFile('https://osf.io/xyqre/download', 'cbc.exe')
```

Comment:
This is the interesting line. It already downloads the solver. The problem is that we need to copy it to PATH-dir or to add the present folder to the PATH. We do not want to execute the solver but to make it available. It will be executet by the python script.

Reply:
What about this python command?

import os
os.environ["PATH"] += os.pathsep + 'C:\\Program Files (x86)\\Graphviz2.38\\bin'

Reply:
https://www.appveyor.com/docs/build-configuration/#installing-additional-software

Reply:
> What about this python command?

Where do you want to put it? I think we need the same command for the power-shell (ps) as shown above.